testng-eclipse-plugin/src/main/org/testng/eclipse/util/CustomSuite.java

```diff
@@ -250,34 +253,30 @@ public File save(File directory) {
     return suiteFile;
   }
 
-  protected void saveSuiteContent(final File file, final XMLStringBuffer content) {
-    FileWriter fw= null;
-    BufferedWriter bw= null;
+  protected void saveSuiteContent(File file, XMLStringBuffer content) {
+    FileOutputStream fileOutputStream = null;
+    BufferedOutputStream bufferedOutputStream = null;
+    OutputStreamWriter osw = null;
     try {
-      fw= new FileWriter(file);
-      bw= new BufferedWriter(fw);
-      bw.write(content.getStringBuffer().toString());
-      bw.flush();
-    }
-    catch(IOException ioe) {
-    }
-    finally {
-      if(null != bw) {
+        fileOutputStream = new FileOutputStream(file);

```

Comment:
for the stream objects, please use 'try-with-resources' statement so that they will be auto-closed